Microsoft.HealthVault/Connection/SessionCredential.cs

```diff
@@ -20,5 +22,10 @@ public class SessionCredential
         /// Gets or sets the shared secret.
         /// </summary>
         public string SharedSecret { get; set; }
+
+        /// <summary>
+        /// Gets or sets the created time of the token.
+        /// </summary>
+        public DateTimeOffset ExpirationUtc { get; set; }
```

Comment:
We should probably make this DateTimeOffset? otherwise you are going to have to compare with the Min value for DateTimeOffset which is ok, but probably not the best. 

---
In reply to: [115827144](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115827144) [](ancestors = 115827144)